ArduPlane/ReleaseNotes.txt

```diff
@@ -1,3 +1,74 @@
+Release 7.0.0 29th November 2024
+------------------------------------------------------------------
+This release includes moving all CxPilot changes on top of ardupilot 4.5.
+
+Differences since CxPilot-5.2.5:
+Fleetwide Changes
+ - Prearm check to ensure all LUA scripts are loaded and running
+ - Improved back transition
+ - JUMP_TAG mission item support
+ - Switching to auto takeoff after manual takeoff no longer adds extra altitude
+ - Wind speed warning reduced to 10m/s (from 12m/s)
+ - Roll limit no longer confined to 7 degrees during transition
+ - FBWB/CRUISE fix massive undershoot issue when switching from max descent to max climb
+ - A whole bunch of bug fixes
+
+Ottano Changes:
+ - Fixed-wing tune improvements. Altitude should be more stable.
+ - Battery assignments standardized
+   1. VTOL batteries combined (reports lowest voltage)
+   2. Avionics battery
+   3. Fuel level sensor
```

Comment:
Maybe note that there is currently no failsafe in place for fuel level until we have a reliable reading on the fuel level sensor.

Reply:
I thought about adding that note, but there's never been a fuel level failsafe has there? If there never has been, I don't want it in the changelog.